spec/lib/populator_spec.rb

```diff
@@ -1,19 +1,31 @@
 describe Populator do
-  describe '.random_ncr_data' do
+  describe '#random_ncr_data' do
     it "creates the specified number of work orders" do
       num_proposals = 5
       expect {
-        Populator.random_ncr_data(num_proposals)
+        Populator.new.random_ncr_data(num_proposals: num_proposals)
       }.to change{ Ncr::WorkOrder.count }.from(0).to(num_proposals)
     end
   end
 
-  describe '.uniform_ncr_data' do
-    it "creates the specified number of work orders" do
-      num_proposals = 5
+  describe '#uniform_ncr_data' do
+    it 'creates the specified number of work orders' do
+      n = 5
       expect {
-        Populator.uniform_ncr_data(num_proposals)
-      }.to change{ Ncr::WorkOrder.count }.from(0).to(num_proposals)
+        Populator.new.uniform_ncr_data(n: n)
+      }.to change{ Ncr::WorkOrder.count }.from(0).to(n)
+    end
+  end
+
+  describe '#ncr_data_for_user' do
+    it 'creates proposals for the user with email address passed in' do
+      email = 'test@example.com'
+      num_proposals = 2
+      user = FactoryGirl.create(:user, email_address: email)
+
+      expect {
+        Populator.new.ncr_data_for_user(email: email, num_proposals: num_proposals)
+      }.to change { user.requested_proposals.count }.from(0).to(num_proposals)
```

Comment:
on second thought, since we are looking at the requested proposals for `user`, we are in effect checking that the requester is the `user`. To be more explicit, we could have 2 expectations - 1 to confirm that 2 records were created, 1 to confirm that the requester for each of those is the user. Thoughts?